compiler/control/OMROptions.cpp

```diff
@@ -1645,6 +1646,9 @@ int32_t       OMR::Options::_trampolineSpacePercentage = 0; // 0 means no change
 bool          OMR::Options::_countsAreProvidedByUser = false;
 TR_YesNoMaybe OMR::Options::_startupTimeMatters = TR_maybe;
 
+int32_t       OMR::Options::_highOccupancyInitialCount = 3000;
```

Comment:
The name does not indicate what kind of occupancy we refer to. Maybe we should call it `_highCodeCacheOccupancyCount`.
Or maybe have a name that indicates a high count without specifying what the count is used for: `highInvocationCount`.
The initial value should be larger than the DEFAULT value which is 3000.

Reply:
Is there an initial value that you had in mind?

Reply:
Maybe 10000